docs/authentication.md

```diff
@@ -28,8 +26,27 @@ The value in the `csrftoken` cookie needs to be passed in an `X-Csrf-Token` HTTP
 A login request, for example, would look like this:
 
 ```bash
-curl 'http://discuit.net/api/_login' -XPOST \
+curl -X POST 'https://discuit.net/api/_login' \
 -H 'Cookie: SID=GyzghHpzr3vOdUG2pOoEeqRBFKwbVWBw5Ovy' \
 -H 'X-Csrf-Token: FcVgW9FZD8w3iptTeh-Nm3cWm4QjVXYulKjqMWjSJkg=' \
 -d '{"username":"neo","password":"whatever"}'
 ```
+
+
+Note: If you are going to use an `.env` file, make sure to escape your quotation marks for proper variable expansion:
+
+`.env` file:  
+
+```ini
+MY_USERNAME=discuit
+MY_PASSWORD=This is a mysterious password!
+```
+
+curl command:  
+
+```bash
+curl -X POST 'https://discuit.net/api/_login' \
+-H 'Cookie: SID=GyzghHpzr3vOdUG2pOoEeqRBFKwbVWBw5Ovy' \
+-H 'X-Csrf-Token: FcVgW9FZD8w3iptTeh-Nm3cWm4QjVXYulKjqMWjSJkg=' \
+-d '{"username":"'"$MY_USERNAME"'","password":"'"$MY_PASSWORD"'"}'
+```
```

Comment:
I don't think we need to add the `.env` file part because the API isn't really meant to be used with `curl`, that's just there as an example. I'd like to add examples in other languages too.

Reply:
It can be if that's what your application uses, but having examples in different languages would probably be more helpful. Plus, I think people who use `.env` files can probably figure this out, so I don't see a point in adding it explicitly to the docs.

Reply:
Okay.  So...close this and just create an issue?